lib/internal/Magento/Framework/Cache/Test/Unit/XsdTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\Cache\Test\Unit;
+
+class XsdTest extends \PHPUnit\Framework\TestCase
```

Comment:
Hi @andrewbess,

I have discovered that we already have bunch of tests which test *.xsd files.
They are located in Magento\Test\Integrity namespace.
Could I ask you to move your your tests under this namespace to keep consistency in tests structure?
Please check \Magento\Test\Integrity\Magento\Widget\WidgetConfigTest for example.

Reply:
Hello @swnsma.
I have moved tests by the suggested path.
Also, I added declaration strict types where it is required.
Please recheck.
Thank you in advance.